eclipselink/src/test/kotlin/com/linecorp/kotlinjdsl/eclipselink/integration/querydsl/select/EclipselinkCriteriaQueryDslSelectIntegrationTest.kt

```diff
@@ -0,0 +1,19 @@
+package com.linecorp.kotlinjdsl.eclipselink.integration.querydsl.select
+
+import com.linecorp.kotlinjdsl.test.integration.EntityManagerExtension
+import com.linecorp.kotlinjdsl.test.integration.querydsl.select.AbstractCriteriaQueryDslSelectIntegrationTest
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import javax.persistence.EntityManager
+
+@ExtendWith(EntityManagerExtension::class)
+class EclipselinkCriteriaQueryDslSelectIntegrationTest : AbstractCriteriaQueryDslSelectIntegrationTest() {
+    override lateinit var entityManager: EntityManager
+
+    @Test
+    override fun `listQuery - select single subquery`() {
+        // https://wiki.eclipse.org/EclipseLink/UserGuide/JPA/Basic_JPA_Development/Querying/JPQL#Sub-selects_in_FROM_clause
+        // eclipselink does not support select subquery and jpa criteria does not support from subquery
```

Comment:
--- korean
select subquery 및 from subquery 보다 subquery in select, subquery in from 이라고 적는 것은 어떤가요?

--- english
Why don't you write "subquery in select, subquery in from" rather than "select subquery and from subquery"?

Reply:
-- korean
이름 반영했습니다.

-- english
name changed

Reply:
-- korean
주석도 수정되면 좋을 것 같습니다.

-- english
It would be good if the comments are changed.

Reply:
-- korean
주석을 수정하다가 고민이 된 부분이 있습니다. 이 테스트는 이렇게 주석으로 테스트를 수행하지 않고 끝내는게 맞을까요 아니면 오류가 나는 상황을 assertion 하는게 나을까요? (예외를 catch 해서 예외 타입을 확인하는 것을 의미합니다). 코멘트는 반영했습니다.

-- english
There was a part where I was troubled while editing the comments. Is it right to end this test without running the test with comments like this, or is it better to assertion an error situation? (means catching the exception and checking the exception type). comments are applied

Reply:
-- korean
예외를 검증하면 eclipselink가 앞으로 지원을 해주는지 여부를 검증할 수 있겠네요.
말씀해주신 것처럼 예외를 assertion 하는 것이 더 좋을 것 같네요. 

-- english
If we verify the exception, we can verify whether eclipselink will support subquery in the future.
As you said, it would be better to verfy exceptions.

Reply:
-- korean
저도 같은 생각으로 문의 드렸습니다. 테스트 반영 하였습니다.

-- english
I also inquired with the same thought. Test reflected.